packages/ember-runtime/lib/mixins/observable.js

```diff
@@ -352,7 +352,7 @@ export default Mixin.create({
     you do not pass a `context` parameter:
 
     ```javascript
-    fooDidChange: function(sender, key, value, rev) { };
```

Comment:
Can you wrap the examples in an object? I'm guessing this minimal code sample might be confusing.